### PR TITLE
[misc] remove is_cuda_available

### PR DESCRIPTION
**Pull request by Alcanderian**
## Motivation

Unify plaform codes

## Checklist

- [ ] Format your code according to the [Code Formatting with Pre-Commit](https://docs.sglang.ai/references/contribution_guide.html#code-formatting-with-pre-commit).
- [ ] Add unit tests as outlined in the [Running Unit Tests](https://docs.sglang.ai/references/contribution_guide.html#running-unit-tests-adding-to-ci).
- [ ] Update documentation / docstrings / example tutorials as needed, according to [Writing Documentation](https://docs.sglang.ai/references/contribution_guide.html#writing-documentation-running-docs-ci).
- [ ] Provide throughput / latency benchmark results and accuracy evaluation results as needed, according to [Benchmark and Profiling](https://docs.sglang.ai/references/benchmark_and_profiling.html) and [Accuracy Results](https://docs.sglang.ai/references/accuracy_evaluation.html).
- [ ] For reviewers: If you haven't made any contributions to this PR and are only assisting with merging the main branch, please remove yourself as a co-author when merging the PR.
- [ ] Please feel free to join our Slack channel at https://slack.sglang.ai to discuss your PR.

### CODE REVIEW

**Review by zhyncs — APPROVED** [review]

File: python/sglang/srt/layers/activation.py [hunk 1]
```diff
@@ -21,9 +21,9 @@
 import torch.nn as nn
 import torch.nn.functional as F
 
-from sglang.srt.utils import is_cuda_available
+from sglang.srt.utils import is_cuda
 
-_is_cuda = is_cuda_available()
```
Comment (Alcanderian) [inline python/sglang/srt/layers/activation.py]: Mark: equivalent to is_cuda

File: python/sglang/srt/layers/attention/triton_ops/double_sparsity_attention.py [hunk 2]
```diff
@@ -3,10 +3,10 @@
 import triton.language as tl
 
 from sglang.srt.managers.schedule_batch import global_server_args_dict
-from sglang.srt.utils import is_hip
+from sglang.srt.utils import is_cuda, is_hip
 
-is_cuda_available = torch.cuda.is_available()
-if is_cuda_available:
```
Comment (Alcanderian) [inline python/sglang/srt/layers/attention/triton_ops/double_sparsity_attention.py:9]: Mark: equivalent to is_cuda because it is used after _is_hip

File: python/sglang/srt/layers/attention/triton_ops/extend_attention.py [hunk 3]
```diff
@@ -23,10 +23,10 @@
 from sglang.srt.layers.attention.triton_ops.prefill_attention import (
     context_attention_fwd,
 )
-from sglang.srt.utils import is_hip
+from sglang.srt.utils import is_cuda, is_hip
 
-is_cuda_available = torch.cuda.is_available()
-if is_cuda_available:
```
Comment (Alcanderian) [inline python/sglang/srt/layers/attention/triton_ops/extend_attention.py:29]: Mark: equivalent to is_cuda because it is used after _is_hip

File: python/sglang/srt/layers/attention/triton_ops/prefill_attention.py [hunk 4]
```diff
@@ -22,8 +22,10 @@
 import triton
 import triton.language as tl
 
-is_cuda_available = torch.cuda.is_available()
-if is_cuda_available:
```
Comment (Alcanderian) [inline python/sglang/srt/layers/attention/triton_ops/prefill_attention.py:26]: Maybe NOT equivalent

File: python/sglang/srt/layers/layernorm.py [hunk 5]
```diff
@@ -19,9 +19,9 @@
 import torch
 import torch.nn as nn
 
-from sglang.srt.utils import is_cuda_available
+from sglang.srt.utils import is_cuda
 
-_is_cuda = is_cuda_available()
```
Comment (Alcanderian) [inline python/sglang/srt/layers/layernorm.py:25]: Mark: equivalent to is_cuda

File: python/sglang/srt/layers/quantization/modelopt_quant.py [hunk 6]
```diff
@@ -22,9 +22,9 @@
     requantize_with_max_scale,
 )
 from sglang.srt.layers.radix_attention import RadixAttention
-from sglang.srt.utils import is_cuda_available
+from sglang.srt.utils import is_cuda
 
-if is_cuda_available():
```
Comment (Alcanderian) [inline python/sglang/srt/layers/quantization/modelopt_quant.py:27]: Mark: equivalent to is_cuda

File: python/sglang/srt/layers/quantization/w8a8_int8.py [hunk 7]
```diff
@@ -2,9 +2,9 @@
 
 import torch
 
-from sglang.srt.utils import is_cuda_available, set_weight_attrs
+from sglang.srt.utils import is_cuda, set_weight_attrs
 
-is_cuda = is_cuda_available()
```
Comment (Alcanderian) [inline python/sglang/srt/layers/quantization/w8a8_int8.py]: Mark: equivalent to is_cuda

File: python/sglang/srt/layers/rotary_embedding.py [hunk 8]
```diff
@@ -8,10 +8,10 @@
 import torch.nn as nn
 
 from sglang.srt.custom_op import CustomOp
-from sglang.srt.utils import is_cuda_available
+from sglang.srt.utils import is_cuda
 
-_is_cuda_available = is_cuda_available()
```
Comment (Alcanderian) [inline python/sglang/srt/layers/rotary_embedding.py:13]: Mark: equivalent to is_cuda

File: python/sglang/srt/layers/sampler.py [hunk 9]
```diff
@@ -10,9 +10,9 @@
 from sglang.srt.layers.logits_processor import LogitsProcessorOutput
 from sglang.srt.managers.schedule_batch import global_server_args_dict
 from sglang.srt.sampling.sampling_batch_info import SamplingBatchInfo
-from sglang.srt.utils import crash_on_warnings, get_bool_env_var, is_cuda_available
+from sglang.srt.utils import crash_on_warnings, get_bool_env_var, is_cuda
 
-if is_cuda_available():
```
Comment (Alcanderian) [inline python/sglang/srt/layers/sampler.py:15]: Mark: equivalent to is_cuda

File: python/sglang/srt/models/minicpm3.py [hunk 10]
```diff
@@ -40,9 +40,9 @@
 from sglang.srt.managers.schedule_batch import global_server_args_dict
 from sglang.srt.model_executor.forward_batch_info import ForwardBatch
 from sglang.srt.model_loader.weight_utils import default_weight_loader
-from sglang.srt.utils import add_prefix, is_cuda_available
+from sglang.srt.utils import add_prefix, is_cuda
 
-if is_cuda_available():
```
Comment (Alcanderian) [inline python/sglang/srt/models/minicpm3.py:45]: Mark: equivalent to is_cuda

File: python/sglang/srt/speculative/build_eagle_tree.py [hunk 11]
```diff
@@ -4,9 +4,9 @@
 
 import torch
 
-from sglang.srt.utils import is_cuda_available, is_hip
+from sglang.srt.utils import is_cuda, is_hip
 
-if is_cuda_available() or is_hip():
```
Comment (Alcanderian) [inline python/sglang/srt/speculative/build_eagle_tree.py:9]: Mark: equivalent to is_cuda

File: python/sglang/srt/speculative/eagle_utils.py [hunk 12]
```diff
@@ -19,9 +19,9 @@
 from sglang.srt.mem_cache.memory_pool import TokenToKVPoolAllocator
 from sglang.srt.model_executor.forward_batch_info import CaptureHiddenMode
 from sglang.srt.speculative.build_eagle_tree import build_tree_kernel_efficient
-from sglang.srt.utils import fast_topk, is_cuda_available, is_hip, next_power_of_2
+from sglang.srt.utils import fast_topk, is_cuda, is_hip, next_power_of_2
 
-if is_cuda_available():
```
Comment (Alcanderian) [inline python/sglang/srt/speculative/eagle_utils.py:24]: Mark: equivalent to is_cuda

File: python/sglang/srt/speculative/eagle_worker.py [hunk 13]
```diff
@@ -34,14 +34,9 @@
...
 from sglang.srt.speculative.spec_info import SpeculativeAlgorithm
-from sglang.srt.utils import (
-    empty_context,
-    fast_topk,
-    get_available_gpu_memory,
-    is_cuda_available,
-)
+from sglang.srt.utils import empty_context, fast_topk, get_available_gpu_memory, is_cuda
 
-if is_cuda_available():
```
Comment (Alcanderian) [inline python/sglang/srt/speculative/eagle_worker.py:44]: Mark: equivalent to is_cuda

**Review by zhyncs — APPROVED** [review]

**Review by merrymercy — APPROVED** [review]
